buildspec.yml

```diff
@@ -0,0 +1,66 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      # install openjdk-8
+      - apt-get update
+      - apt-get -y install openjdk-8-jdk
+      - update-java-alternatives -s java-1.8.0-openjdk-amd64
+      - export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/bin
+
+      # install sbt launcher
+      - curl -LO https://github.com/sbt/sbt/releases/download/v1.1.6/sbt-1.1.6.tgz
+      - tar -xf sbt-*.tgz
+      - export PATH=$CODEBUILD_SRC_DIR/sbt/bin/:$PATH
+      - cd $CODEBUILD_SRC_DIR/sagemaker-spark-sdk
+      - sbt -Dsbt.log.noformat=true sbtVersion scalaVersion
+
+  pre_build:
+    commands:
+      # clone the integration test repo
+      - cd $CODEBUILD_SRC_DIR
+      - GITHUB_OAUTH_TOKEN=$(aws secretsmanager get-secret-value --secret-id /codebuild/github/oauth --query SecretString --output text)
+      - INTEGRATION_TEST_REPO_URI="https://${GITHUB_OAUTH_TOKEN}@github.com/jesterhazy/PRIVATE-sagemaker-spark-integ-tests.git"
+      - git clone --depth 1 --single-branch $INTEGRATION_TEST_REPO_URI integration-tests
```

Comment:
Why we cannot consume it as a codebuild source?

Reply:
we could, but then this project would not be able to share the same template or configuration as our other PR builds. this seemed better than setting this project up as a one-off or significant refactoring of the overall PR setup. 

plus, putting the integration tests into a separate project seems like a mistake that we should sort out in a future PR.